SPHINXsys/src/shared/materials/riemann_solver.cpp

```diff
@@ -77,232 +77,198 @@ namespace SPH
 			   e_ij * (u_star - (ul * state_i.rho_ + ur * state_j.rho_) / (state_i.rho_ + state_j.rho_));
 	}
 	//=================================================================================================//
-	Real HLLCRiemannSolver::
-		getPStar(const CompressibleFluidState &state_i, const CompressibleFluidState &state_j, const Vecd &e_ij)
+	FluidState HLLCRiemannSolverInWeaklyCompressibleFluid::
```

Comment:
@mrezavandVirtonomy please check these fluid parts

Reply:
This part is in developing.